representation/tests/build-logical-test.metta

```diff
@@ -0,0 +1,39 @@
+!(register-module! ../../../metta-moses)
+!(import! &self metta-moses:utilities:tree)
+!(import! &self metta-moses:utilities:list-methods)
+!(import! &self metta-moses:utilities:general-helpers)
+!(import! &self metta-moses:representation:knob-representation)
+!(import! &self metta-moses:representation:logical-probe)
+!(import! &self metta-moses:representation:knob-representation)
+!(import! &self metta-moses:representation:lsk)
+!(import! &self metta-moses:utilities:nodeId)
+!(import! &self metta-moses:representation:sample-logical-perms) 
+!(import! &self metta-moses:representation:add-logical-knobs)
+!(import! &self metta-moses:utilities:ordered-multimap) 
+!(import! &self metta-moses:representation:build-logical)
+! (import! &self metta-moses:utilities:python-treehelpers)
+
+(= (ARGS) args) 
+(= (APPEND_CHILD $tree $nodeId $child ) (py_appendChild $tree $nodeId $child))
+(= (GetByID $tree $nodeId) (py_getById $tree $nodeId))
+
+(= (INSERT_ABOVE $tree $nodeId $subtree) (py_insertAbove  $tree $nodeId $subtree))
+
+
+!(bind! tree1 (mkTree (mkNode AND) 
+                  (Cons (mkTree (mkNode A) Nil) 
+                  (Cons (mkTree (mkNode B) Nil) 
+                  (Cons (mkTree (mkNode OR) 
+                        (Cons (mkTree (mkNode C) Nil) Nil)) 
+                        Nil)))))
+;; buildLogical test case
```

Comment:
Can you please add comments for what this test case is? Like the input and it's expected outcome...